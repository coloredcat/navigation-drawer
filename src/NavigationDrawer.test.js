import React from 'react'
import { configure, shallow, mount } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import renderer from 'react-test-renderer'
import SideMenu, { Controls } from './NavigationDrawer'

configure({ adapter: new Adapter() })

// Snapshot for Home React Component
describe('>>>Sidemenu --- Snapshot', () => {
  it('+++capturing Snapshot of Sidemenu', () => {
    const wrapper = mount(
      <SideMenu>
        <Controls />
      </SideMenu>
    )
    expect(wrapper.html()).toMatchSnapshot()
  })
})
