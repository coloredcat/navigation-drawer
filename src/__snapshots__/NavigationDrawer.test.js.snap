// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation Drawer creates new snapshot 1`] = `"<div class=\\"sc-dlnjPT eFWNyw\\"><div id=\\"drawer__root\\" style=\\"padding-left: 60px; display: flex; transform: translateX(calc(-360px));\\" class=\\"sc-hKFyIo jKpTHc\\"><div class=\\"sc-eCApGN bdszxu\\"><ul class=\\"sc-jSFkmK gdvzsZ\\">Stuff</ul></div></div><div id=\\"app__root\\" style=\\"height: 100%;\\"><div class=\\"sc-gtssRu iGNmzI\\"><button>Deactivate drawer</button></div></div></div>"`;

exports[`Navigation Drawer renders without errors 1`] = `
<div>
  <div
    class="sc-dlnjPT eFWNyw"
  >
    <div
      class="sc-hKFyIo jKpTHc"
      id="drawer__root"
      style="padding-left: 60px; display: flex; transform: translateX(calc(-360px));"
    >
      <div
        class="sc-eCApGN bdszxu"
      >
        <ul
          class="sc-jSFkmK gdvzsZ"
        >
          Stuff
        </ul>
      </div>
    </div>
    <div
      data-testid="app__root"
      style="height: 100%;"
    >
      <div
        class="sc-gtssRu iGNmzI"
      >
        <button>
          Deactivate drawer
        </button>
      </div>
    </div>
  </div>
</div>
`;
